js/hubs/components/ReleaseCreateConfirm.js

```diff
@@ -109,14 +115,22 @@ const ReleaseCreateConfirm = (props) => {
             <Value className="description" sx={{mt: 1, flexDirection: 'column' , mb: 1}}>
               Description: <span style={{marginTop: '8px', paddingLeft: '0'}}>{description}</span>
             </Value>
+            <Typography variant="subtitle1" mt={1} sx={{ color: "red" }}>
+              ONCE PUBLISHED, YOUR RELEASE INFORMATION WILL BE PERMANENT AND YOU
+              WILL NOT BE ABLE TO EDIT IT.
+            </Typography>
+            <Value>
+              <FormControlLabel sx={{ mt: 1, mb: 1, paddingLeft: '10px' }} control={<Checkbox onChange={handleChangeCheckbox} />} label="Confirm" />
```

Comment:
we should kill set padding on the `span` around the input rendered in here to make it align w the rest of the elements